README.md

```diff
@@ -16,12 +16,15 @@
 
 ## Table of Contents
 
-- [Install](#install)
-- [Usage](#usage)
-- [Supported Hash Types](#supported-hash-types)
-- [Maintainers](#maintainers)
-- [Contribute](#contribute)
-- [License](#license)
+- [rust-multihash](#rust-multihash)
```

Comment:
I think the README is small enough to keep the current table of contents. What do you think @boneyard93501?

Reply:
absolutely. for some reason, vscode likes to rearrange the list and i didn't notice.
```
- [Table of Contents](#table-of-contents)
  - [Install](#install)
  - [Usage](#usage)
      - [rust-multihash](#rust-multihash)
      - [Using a custom code table](#using-a-custom-code-table)
  - [Supported Hash Types](#supported-hash-types)
  - [Maintainers](#maintainers)
  - [Contribute](#contribute)
  - [License](#license)
``` 
work>